src/comm.cpp

```diff
@@ -126,6 +128,21 @@ void greet(Descriptor &d) {
     d.write(greeting->text());
 }
 
+namespace {
+bool is_being_debugged() {
+    std::ifstream t("/proc/self/status");
```

Comment:
aha yes, I'd read about this :)

Reply:
There ought to be a standard API for this sort of thing like in Windows.

Reply:
Yeah it's a pain in the proverbials...this is the best I could find.